tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -5241,6 +5241,7 @@ def globalReadGuardKBody(tP):
       tcDataType = "" if tc == "Metadata" else tc
       graIdx = 0
       g2lIdx = 0
+      bpeRatio  = tP["bpe"] // tP["bpeGR"] if tP["bpeGR"] < tP["bpe"] else 1
```

Comment:
calculate it in getTensorParameters() stage?

Reply:
Done